output/md/md.go

```diff
@@ -129,6 +129,11 @@ func (m *Md) OutputViewpoint(wr io.Writer, i int, v *schema.Viewpoint) error {
 
 // Output generate markdown files.
 func Output(s *schema.Schema, c *config.Config, force bool) (e error) {
+	s, err := s.SetViewpointsToTables()
```

Comment:
I'm not quite sure if this is is appropriate to set viewpoints to table.
I could be better to be in config.Analyze but it can effect more files.

Reply:
Let's start with the minimum effect range.

I like refactoring 👍